kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -52,6 +65,10 @@ define([
             this.fileStagingClient = new FileStaging(
```

Comment:
We should pick one Staging Service client and roll with it. We both rolled our own. I gotta say, I like yours better because it comes with a more generic REST client that we could use elsewhere.